ras_party/controllers/respondent_controller.py

```diff
@@ -78,7 +78,7 @@ def change_respondent_details(respondent_data, respondent_id, session):
 
     respondent = query_respondent_by_party_uuid(respondent_id, session)
     if not respondent:
-        raise ClientError("Respondent id does not exist", respondent_id=respondent_id, status=404)
+        raise ClientError("Respondent id does not exist", respondent_id=str(respondent_id), status=404)
```

Comment:
`respondent_id ` comes from the url, so should already be a string

Reply:
@ajmaddaford  All done.